test/int/deployment_test.go

```diff
@@ -490,7 +520,7 @@ var _ = Describe("AtlasDeployment", Label("int", "AtlasDeployment"), func() {
 			})
 
 			By("Decreasing the Deployment disk size", func() {
-				createdDeployment.Spec.DeploymentSpec.DiskSizeGB = intptr(10)
+				createdDeployment.Spec.DeploymentSpec.DiskSizeGB = intptr(15)
```

Comment:
I don't think we set `DiskSizeGB` for the `DefaultAWSDeployment` which probably means it is 10 (which is the minimum accepted value). Which probably means this line is actually increases the `DiskSizeGB`.

Reply:
I changed the default disk size to 20GB, but It looks like I forgot to add this file, thanks!